blog/2021/apisix.adoc

```diff
@@ -0,0 +1,184 @@
+:title: How to Integrate Keycloak for Authentication with Apache APISIX
+:date: 2021-12-13
+:publish: true
+:author: Xinxin Zhu & Yilin Zeng
+
+image::${blogImages}/apisix/keycloak-apisix-integration.png[keycloak-apisix-integration]
```

Comment:
I'd propose removing this image, or making it a lot less tall. Right now I have to scroll quite a bit before seeing any text. Happy to keep it as is if you prefer that though.

Reply:
> I'd propose removing this image, or making it a lot less tall. Right now I have to scroll quite a bit before seeing any text. Happy to keep it as is if you prefer that though.

Yes, the image is quite tall for the page. Removed now.